examples/sasl_producer.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
```

Comment:
Since this is to show-case how to set up SASL and the SASL setups are identical I don't really see the need for both a consumer and producer.
If we were to keep one I suggest keeping the producer since it does not require another party to operate.